services/ml/compilation_delegate_bnns.mm

```diff
@@ -525,6 +527,49 @@ void SetExtendInputs(const uint32_t extend_input_idx,
   return true;
 }
 
+bool CompilationDelegateBnns::CompileLogistic(
+    const mojom::ModelInfoPtr& model,
+    const mojom::OperationPtr& operation,
+    OperationMac& operation_bnns) {
+  DLOG(INFO) << "CompilationDelegateBnns::CompilLogistic";
+
+  operation_bnns.offset_x = 0;
+  operation_bnns.offset_y = 0;
+
+  BNNSVectorDescriptor in_desc, out_desc;
+  const std::vector<uint32_t>& inputs = operation->inputs;
+  const std::vector<uint32_t>& outputs = operation->outputs;
+  const mojom::OperandPtr& input = model->operands[inputs[0]];
+  const mojom::OperandPtr& output = model->operands[outputs[0]];
+
+  operation_bnns.input_batch_size = input->dimensions[0];
+
+  int32_t size = 1;
+  for (size_t i = 1; i < input->dimensions.size(); i++) {
```

Comment:
We have common function `product(input->dimensions)`

Reply:
The loop is from the index 1.